cv32e40p/sim/Common.mk

```diff
@@ -21,7 +21,7 @@ CV_CORE_TAG    ?= cv32e40p_v1.0.0
 
 RISCVDV_REPO    ?= https://github.com/google/riscv-dv
 RISCVDV_BRANCH  ?= master
-RISCVDV_HASH    ?= 10fd4fa8b7d0808732ecf656c213866cae37045a
+RISCVDV_HASH		?= 0b625258549e733082c12e5dc749f05aefb07d5a
```

Comment:
@silabs-halfdan, just FYI, you can use the most significant seven hex-digits when referencing a git hash.  So this could be `RISCVDV_HASH ?= 0b62525`.  (I learned this several weeks after generating the previous version of this Makefile.)  The full hash is only required in the event of a collision.